website/docs/r/schema_template_vrf_contract.html.markdown

```diff
@@ -13,15 +13,17 @@ Manages MSO Resource Schema Template Vrf Contract. This resource is supported in
 ## Example Usage ##
 
 ```hcl
+
 resource "mso_schema_template_vrf_contract" "demovrf01" {
-  schema_id              = "5eff091b0e00008318cff859"
+  schema_id              = mso_schema.schema1.id
   template_name          = "Template1"
   vrf_name               = "myVrf"
   relationship_type      = "provider"
   contract_name          = "hubcon"
```

Comment:
should point to mso_schema_template_contract

Reply:
ok